src/envdrift/cli_commands/vault.py

```
@@ -50,37 +59,150 @@ def vault_push(
+                    print_error(f"Skipped {mapping.folder_path}: {key_name} not found in keys file")
+                    error_count += 1
+                    continue
+
+                actual_value = f"{key_name}={key_value}"
+
+                # Push
+                client.set_secret(mapping.secret_name, actual_value)
+                print_success(f"Pushed {mapping.secret_name}")
+                pushed_count += 1
+
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)